lib/application/widgets/theme/themed_tab_bar.dart

```diff
@@ -0,0 +1,36 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_hooks/flutter_hooks.dart';
+import 'package:memo/application/theme/theme_controller.dart';
+import 'package:memo/application/constants/dimensions.dart' as dimens;
+
+/// Decorates a [TabBar] with custom layout specs
+class ThemedTabBar extends HookWidget {
+  const ThemedTabBar({required this.controller, required this.tabs, Key? key}) : super(key: key);
+
+  final TabController controller;
+  final List<Widget> tabs;
+
+  @override
+  Widget build(BuildContext context) {
+    final memoTheme = useTheme();
+    final selectedColor = memoTheme.secondarySwatch.shade400;
+    final unselectedColor = memoTheme.neutralSwatch.shade700;
+    const borderHeight = dimens.tabBarBorderHeight;
+
+    final tabBar = TabBar(
+      controller: controller,
+      tabs: tabs,
+      indicatorWeight: borderHeight * 2,
```

Comment:
Is this condition always true? I mean, it seems like an arbitrary multiplication. If `dimens.tabBarBorderHeight` changes for some reason, this expression keeps valid?

Reply:
yes because this is double the back indicator height